styles/globals.css

```diff
@@ -33,7 +33,7 @@ a {
   text-decoration: none;
 }
 
-#__next {
+body {
   height: 100%;
```

Comment:
10x for making a contribution @AvielBitton.
There is already a body selector with a height of 100% on the selectors.
moreover, the specific body selector should be near the existing body selector on that file.
could you please change the above?

Reply:
hi @Yaronglp , thanks for your comment; it was my honor to contribute to this project. 
I push fix changes